### DISCUSSION TITLE
MicroSD not detected

### DISCUSSION THREAD

**Discussion by dedvoltg** [erichelgeson/BlueSCSI-v1#188] [Q&A] [answered]
So I recently built a blue SCSI and when I first tried it out and gave me the five flashing lights which obviously means no SD card I went through and changed the file system from exFat to FAT32 and nothing worked. After that, I reflashed the blue pill then I tested all of the traces to make sure there was continuity still nothing I made sure to get a really good look at everything to make sure everything was Soldered correctly and also double-checked the positions and everything but I nothing what else could I be doing wrong?
![20221016_153332](https://user-images.githubusercontent.com/93351051/196054831-b1238115-83c6-4496-a579-431c52aea84e.jpg)
![20221016_153248](https://user-images.githubusercontent.com/93351051/196054833-c8eb9f8b-f577-4863-962e-bb9bd184f793.jpg)
![20221016_153238](https://user-images.githubusercontent.com/93351051/196054834-87529db3-480f-40f4-9d9f-4e4e5f8d8080.jpg)
![20221016_153344](https://user-images.githubusercontent.com/93351051/196054835-2c6d6e87-6abb-450d-89fd-a87403d35585.jpg)

**Comment by dedvoltg** [comment, marked as answer]
I just fixed it myself by using the SD formatting tool that you can find in the troubleshooting tab.

**Reply by erichelgeson** [reply]
Awesome! Glad you got it going. Have fun!
